### ISSUE TITLE
accessibility: how do we deal with tab navigation through table elements

### ISSUE THREAD

**Issue by cwyckoff3** [kubernetes-sigs/kui#2507]
Known Accessibility violations in DAP(Dynamic Assessment Plugin): 

Each form control must have associated label: Form control **input** is missing associated label. [Link](https://aat.w3ibm.mybluemix.net/token/d35aa089-c3ab-4fc7-8b39-af9d587b36f3/44d1cd83-670f-4469-aa66-51e313fd0861/archives/2019JulyDeploy//doc/w3/help/en-US/idhi_accessibility_check_g41.html)
- [x] Each time a command is run
- [x] "enter your command" placeholder text in entry box

[Link](https://aat.w3ibm.mybluemix.net/token/d35aa089-c3ab-4fc7-8b39-af9d587b36f3/44d1cd83-670f-4469-aa66-51e313fd0861/archives/2019JulyDeploy//doc/w3/help/en-US/idhi_accessibility_check_g490.html)
- [x] Page must identify the default language of the document with a lang attribute

Contrast of foreground and background must be sufficient to meet WCAG AA requirements. Need 4.5 contrast [Link](https://aat.w3ibm.mybluemix.net/token/d35aa089-c3ab-4fc7-8b39-af9d587b36f3/44d1cd83-670f-4469-aa66-51e313fd0861/archives/2019JulyDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1148.html)
- [x] time displayed when a command is run, 3.51 contrast
- [x] With Dark theme on,  Header text: "Tab", productName(themes.json) text
- [x] "ok" text returned when a command finishes. Ex: about

An element with WAI-ARIA role must be contained within a valid element: element with WAI-ARIA role tab is not contained in or owned by an element with WAI-ARIA role: tablist  [Link](https://aat.w3ibm.mybluemix.net/token/d35aa089-c3ab-4fc7-8b39-af9d587b36f3/44d1cd83-670f-4469-aa66-51e313fd0861/archives/2019JulyDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1153.html)
- [x] Tabs in sidecar along with "Delete this resource"
- [x] Tabs in about page

Images must have an alt attribute, and must be empty string if decorative
- [x] About tab image 

Keyboard Operations:
- [ ] Table results (kubectl get pods) aren't accessible via tab
- [x] About sidecar cannot be accessed via tab
- [x] Detailed sidecar tabs cannot be accessed

**Comment by starpit** [comment]
about image alt covered by #2520

**Comment by starpit** [comment]
covered by #2594:

About sidecar cannot be accessed via tab
Detailed sidecar tabs cannot be accessed

**Comment by myan9** [comment]
> ok" text returned when a command finishes. Ex: about

^^^ covered by https://github.com/IBM/kui/pull/2591

**Comment by cwyckoff3** [comment]
@myan9  A few small violations found after rerunning scan..
"Hyperlinks must contain link text or an image with alt text" 
- New tab button "+"
- Maximize, close, and sign out buttons in the sidecar

**Comment by starpit** [comment]
i'm reopening this. the #3076 fix is a potential fix for the table tab navigation, but i think it needs more consideration: if there are dozens of tables in the repl scroll buffer, each with a dozen rows... do we want to force hundreds of tabs to cycle through the UI elements?
